diff --git a/app/code/core/Mage/Core/Helper/Data.php b/app/code/core/Mage/Core/Helper/Data.php
index b9bf867..2a94a96 100644
--- a/app/code/core/Mage/Core/Helper/Data.php
+++ b/app/code/core/Mage/Core/Helper/Data.php
@@ -1052,15 +1052,10 @@ XML;
      */
     public function isValidLength(string $value, ?int $min = null, ?int $max = null): bool
     {
-        $options = [];
-        if ($min !== null) {
-            $options['min'] = $min;
-        }
-        if ($max !== null) {
-            $options['max'] = $max;
-        }
-
-        $violations = $this->getSymfonyValidator()->validate($value, new Assert\Length($options));
+        $violations = $this->getSymfonyValidator()->validate(
+            $value,
+            new Assert\Length(min: $min, max: $max),
+        );
         return count($violations) === 0;
     }
 
