diff --git a/lib/Maho/Data/Collection.php b/lib/Maho/Data/Collection.php
--- a/lib/Maho/Data/Collection.php
+++ b/lib/Maho/Data/Collection.php
@@ -350,9 +350,6 @@
         $itemId = $this->_getItemId($item);
 
         if (!is_null($itemId)) {
-            if (isset($this->_items[$itemId])) {
-                throw new \Exception('Item (' . $item::class . ') with the same id "' . $item->getId() . '" already exist');
-            }
             $this->_items[$itemId] = $item;
         } else {
             $this->_addItem($item);
